### PR TITLE
gcp-lts-bom: adding google-cloud-iamcredentials

### PR DESCRIPTION
**Pull request by suztomo**
@chingor13 @Neenu1995 As discussed in the chat, adding the google-cloud-iamcredentials artifact in the BOM.

### CODE REVIEW

File: boms/cloud-lts-bom/pom.xml [hunk 1]
```diff
@@ -267,8 +268,17 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <artifactId>google-iam-admin</artifactId>
+        <artifactId>google-iam-admin-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:271]: Using the BOM to cover API surface.

File: boms/cloud-lts-bom/pom.xml [hunk 2]
```diff
@@ -267,8 +268,17 @@
...
         <version>${google-iam-admin.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-iamcredentials-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:278]: Using the BOM to cover API surface.

File: boms/cloud-lts-bom/pom.xml (hunk 1 above)
Comment (dzou) [reply boms/cloud-lts-bom/pom.xml:271]: Sorry, just to clarify, what's the reason for putting bom here instead of standard dependencies?

**Review by dzou — APPROVED** [review]

File: boms/cloud-lts-bom/pom.xml (hunk 1 above)
Comment (suztomo) [reply boms/cloud-lts-bom/pom.xml:271]: It's to cover API surface.

**Pull request merged**